sharding/indexHashedNodesCoordinatorRegistry.go

```diff
@@ -4,14 +4,16 @@ import (
 	"encoding/json"
 	"fmt"
 	"strconv"
+
+	"github.com/ElrondNetwork/elrond-go/core"
 )
 
 const keyPrefix = "indexHashed_"
 
 // SerializableValidator holds the minimal data required for marshalling and un-marshalling a validator
 type SerializableValidator struct {
```

Comment:
Please add a TODO to refactor this structure so it will work with gogo proto marshalizer

Reply:
done